CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing to NetPanzer
+
+## Bug Reports and Feature Requests
+
+Anyone may [open an
+issue](https://github.com/netpanzer/netpanzer/issues).
+
+## Discussion
+
+If you're not sure if something is a bug, you can search, or post on, the
+[Discussions](https://github.com/netpanzer/netpanzer/issues).
+
+### Discord
+
+You may participate on our [Discord server](https://discord.gg/kCAB2CMswd).
+It's important to remember that any discussions about bugs or desired features
+in a chat room usually scroll away and become forgotten, so it's best to
+create a ticket or discussion on GitHub (mentioned above) if you really want a
+bug fix or new feature.
+
+## Coding Standards
```

Comment:
@winrid At least this section needs to be completed before merging. I don't know what this should be.

Reply:
for sure - I approved since it was an improvement over what we have today, okay to merge it :) we can probably copy something from another project. I don't want to be too strict.